loader/jni_patch.c

```diff
@@ -110,6 +110,15 @@ int GetGamepadButtons(int port) {
 
   int mask = 0;
 
+  if (pad.buttons & SCE_CTRL_SELECT)
```

Comment:
Instead of returning 0, please move this back down and instead do mask & ~(0x20 | 0x40) when both are pressed...